### ISSUE TITLE
Cannot import cv2 in a fresh install of py-opencv

### ISSUE THREAD

**Issue by ocefpaf** [conda-forge/opencv-feedstock#202]
Looks like the `cv2` module lives in `py-open`

https://github.com/conda-forge/opencv-feedstock/blob/9b6581f1b5302e4d0ad819c55287cf82207a5bbc/recipe/meta.yaml#L209

but installing that with `conda create -n TEST python=3.7 py-opencv` (also tested py38) fails to install the `cv2` module and the import fails. On a related note, installing the metapackage `opencv` itself fails with multiple conflicts on Python 3.8.

**Comment by hmaarrfk** [comment]
Yeah.... I can confirm this is also the case on my machine

**Comment by hmaarrfk** [comment]
Opening up the opencv package on conda-forge, I see that it is pinned to
```
{
  "arch": "x86_64",
  "build": "1",
  "build_number": 1,
  "depends": [
    "libopencv 4.1.2 h392e6d5_1",
    "py-opencv 4.1.2 py27h5ca1d4c_1"
  ],
  "license": "BSD 3-clause",
  "name": "opencv",
  "platform": "linux",
  "subdir": "linux-64",
  "timestamp": 1574260762535,
  "version": "4.1.2"
}
```

The lib opencv package also seems to have been created specifically for the python 2.7 version.


I can also confirm that

 ```
conda create --name py38 python=3.8 opencv
```

doesn't work.

**Comment by hmaarrfk** [comment]
This seems to have been fixed in the 4.x branch. I'll merge in the fix for the 3.4 branch

**Comment by ocefpaf** [comment]
Thanks @hmaarrfk! I believe I restart all the failing py38 PRs that failed b/c of this. I guess there is still one that will need the 3.4 you just merged.
